store/postgres/src/relational.rs

```diff
@@ -54,6 +54,7 @@ const DELETE_OPERATION_CHUNK_SIZE: usize = 1_000;
 /// This also makes sure that we do not put strings into a BTree index that's
 /// bigger than Postgres' limit on such strings which is about 2k
 pub const STRING_PREFIX_SIZE: usize = 256;
+pub const BYTE_ARRAY_PREFIX_SIZE: usize = 256;
```

Comment:
I wonder if we could make that one smaller, say 64, in the interest of keeping indexes smaller since `bytea` should have more entropy than `text`.